### PR TITLE
Changes to readme 

### PR DESCRIPTION
**Pull request by selvasingh**

### CODE REVIEW

File: notes/prepare-for-1.6.0.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Prepare for Azure Management Libraries for Java 1.6.0 #
+
+Steps to migrate code that uses Azure Management Libraries for Java from 1.5 to 1.6 ...
+
+> If this note missed any breaking changes, please open a pull request.
+
+
+V1.6 is backwards compatible with V1.5 in the APIs intended for public use that reached the general availability (stable) stage in V1.x. 
+
+Some breaking changes were introduced in APIs that were still in Beta in V1.5, as indicated by the `@Beta` annotation.
```
Comment (anuchandy) [inline notes/prepare-for-1.6.0.md]: that were still in Beta in V1.6

File: notes/prepare-for-1.6.0.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+
```
Comment (anuchandy) [inline notes/prepare-for-1.6.0.md]: we should remove the following sections as there is no `Renames/Removals` or ` Changes in Input Parameters Order`

File: README.md [hunk 3]
```diff
@@ -656,6 +679,7 @@ If you would like to become an active contributor to this project please follow
 
 | Version           | SHA1                                                                                      | Remarks                                               |
 |-------------------|-------------------------------------------------------------------------------------------|-------------------------------------------------------|
+| 1.6.0       | [1.6.0](https://github.com/Azure/azure-libraries-for-java/tree/v1.6.0)               | Tagged release for 1.6.0 version of Azure management libraries |
```
Comment (anuchandy) [inline README.md]: Missing an entry for 1.5.1

```
| 1.5.1       | [1.5.1](https://github.com/Azure/azure-libraries-for-java/tree/v1.5.1)               | Tagged release for 1.5.1 version of Azure management libraries | 
```

Comment (selvasingh) [reply README.md]: Replaced the 1.5.0 entry with 1.5.1 which is the correct entry

File: notes/prepare-for-1.6.0.md (hunk 2 above)
Comment (selvasingh) [reply notes/prepare-for-1.6.0.md]: Dropped

File: notes/prepare-for-1.6.0.md (hunk 1 above)
Comment (selvasingh) [reply notes/prepare-for-1.6.0.md]: Dropped the second sentence - "Some breaking changes were .. "

**Pull request merged**